CMakeLists.txt

```diff
@@ -1000,6 +1001,13 @@ if (Seastar_NUMA)
     PRIVATE numactl::numactl)
 endif ()
 
+if (SystemTap-SDT_FOUND)
+  list (APPEND Seastar_PRIVATE_COMPILE_DEFINITIONS SEASTAR_HAVE_SYSTEMTAP_SDT)
+
+  target_link_libraries (seastar
+    PRIVATE SystemTap::SDT)
+endif ()
```

Comment:
What library is needed here? AFAIK none.

Reply:
the library is but the header provided by systemtap-sdt-dev. normally it's installed into `/usr/include`. IMHO, to use `find_package()` is just a way to 

1. make it explicit that the library is not a part of libc or system (linux) header
2. `find_path()` searches with a set of predefined rules for finding the header file. the rules allows use to customize the place where the header to be found without changing the building system. for instance, he/she can define `<PackageName>_ROOT` and `find_path()` will search the header in it.